### ISSUE TITLE
Please go to https://panic.apple.com

### ISSUE THREAD

**Issue by jhaotingc** [fidele007/Asus-ROG-GL552VW-Hackintosh#13]
I copied all the CLOVER from github and followed the detailed guide, but I still got this.
![60743047_431518267626933_2251708193442889728_n](https://user-images.githubusercontent.com/13596495/58127550-3595a580-7c48-11e9-8824-5e1a4aea57bd.jpg)
Does anyone know how to fix it?

**Comment by HyRespt** [comment]
i got that issue too 

![20190524_113222_1](https://user-images.githubusercontent.com/30009072/58337997-af4bb000-7e4f-11e9-808d-d863b1a9a578.gif)

 i achieved to freeze the error from video
![VideoCapture_20190524-113636](https://user-images.githubusercontent.com/30009072/58337750-2765a600-7e4f-11e9-8704-874c3fe2d8b0.jpg)

**Comment by jhaotingc** [comment]
> i got that issue too
> 
> ![20190524_113222_1](https://user-images.githubusercontent.com/30009072/58337997-af4bb000-7e4f-11e9-808d-d863b1a9a578.gif)
> 
> i achieved to freeze the error from video
> ![VideoCapture_20190524-113636](https://user-images.githubusercontent.com/30009072/58337750-2765a600-7e4f-11e9-8704-874c3fe2d8b0.jpg)

I passed that issue by download the essential kext from the web(latest), except the ones started with SATA(four of them) I copied from github. After that, I copied config_preinstall.plist from github and rename it to config.plist and them paste it in the USB.
And then I got the new error on the new issue.

**Comment by HyRespt** [comment]
> I copied all the CLOVER from github and followed the detailed guide, but I still got this.
> ![60743047_431518267626933_2251708193442889728_n](https://user-images.githubusercontent.com/13596495/58127550-3595a580-7c48-11e9-8824-5e1a4aea57bd.jpg)
> Does anyone know how to fix it?



> > i got that issue too
> > ![20190524_113222_1](https://user-images.githubusercontent.com/30009072/58337997-af4bb000-7e4f-11e9-808d-d863b1a9a578.gif)
> > i achieved to freeze the error from video
> > ![VideoCapture_20190524-113636](https://user-images.githubusercontent.com/30009072/58337750-2765a600-7e4f-11e9-8704-874c3fe2d8b0.jpg)
> 
> I passed that issue by download the essential kext from the web(latest), except the ones started with SATA(four of them) I copied from github. After that, I copied config_preinstall.plist from github and rename it to config.plist and them paste it in the USB.
> And then I got the new error on the new issue.

can you reach me from telegram (same nickname) or mail (on my profile) I have an idea (about your other issue too) if it works then we will post it here

**Comment by jhaotingc** [comment]
> > I copied all the CLOVER from github and followed the detailed guide, but I still got this.
> > ![60743047_431518267626933_2251708193442889728_n](https://user-images.githubusercontent.com/13596495/58127550-3595a580-7c48-11e9-8824-5e1a4aea57bd.jpg)
> > Does anyone know how to fix it?
> 
> > > i got that issue too
> > > ![20190524_113222_1](https://user-images.githubusercontent.com/30009072/58337997-af4bb000-7e4f-11e9-808d-d863b1a9a578.gif)
> > > i achieved to freeze the error from video
> > > ![VideoCapture_20190524-113636](https://user-images.githubusercontent.com/30009072/58337750-2765a600-7e4f-11e9-8704-874c3fe2d8b0.jpg)
> > 
> > 
> > I passed that issue by download the essential kext from the web(latest), except the ones started with SATA(four of them) I copied from github. After that, I copied config_preinstall.plist from github and rename it to config.plist and them paste it in the USB.
> > And then I got the new error on the new issue.
> 
> can you reach me from telegram (same nickname) or mail (on my profile) I have an idea (about your other issue too) if it works then we will post it here

I cannot add you to the group because of your privacy settings
Add me @jack34672 on telegram

**Comment by ohua28** [comment]
Got same issue i already downloaded the latest kext files

**Comment by baobaoit** [comment]
try my clover: 
[CLOVER.zip](https://github.com/fidele007/Asus-ROG-GL552VW-Hackintosh/files/3264728/CLOVER.zip)

**Comment by pparrilla** [comment]
Hi, I'm a beginner in the hackintosh community. When you say try my Clover.zip, it's copy and paste into the partition directly, right?

**Comment by baobaoit** [comment]
> Hi, I'm a beginner in the hackintosh community. When you say try my Clover.zip, it's copy and paste into the partition directly, right?

Yes, it means you replace your clover folder with my clover folder.

**Comment by do0k** [comment]
> try my clover:
> [CLOVER.zip](https://github.com/fidele007/Asus-ROG-GL552VW-Hackintosh/files/3264728/CLOVER.zip)

hi I used your clover file but it freezes on audio changer something

**Comment by baobaoit** [comment]
> > try my clover:
> > [CLOVER.zip](https://github.com/fidele007/Asus-ROG-GL552VW-Hackintosh/files/3264728/CLOVER.zip)
> 
> hi I used your clover file but it freezes on audio changer something

I think it is different something, you need to redo by your self.

**Comment by oq-x** [comment]
i upgraded my hackintosh big sur to monterey and i have the same problem
can someone help?
